include/cbl/CBLDatabase.h

```diff
@@ -55,10 +52,9 @@ typedef struct CBLEncryptionKey {
 /** Database configuration options. */
 typedef struct {
     FLString directory;                     ///< The parent directory of the database
-    CBLEncryptionKey* encryptionKey;        ///< The database's encryption key (if any)
+    CBLEncryptionKey* encryptionKey;        ///< (Enterprise Edition Only) The database's encryption key (if any)
```

Comment:
This is compiled out of CE builds on other platforms

Reply:
First, I was thinking about putting `ifdef COUCHBASE_ENTERPRISE` there, but then unpleasant code changes (e.g. branching constructor and so on) are needed so I stopped (but that shouldn't be the reason not to do it). 

I'm not quite understand about putting `ifdef COUCHBASE_ENTERPRISE` in the public header files as users will then need a defined COUCHBASE_ENTERPRISE (maybe need to define it in an EE header?). I plan to have some discussion about this when doing EE repo. Let's chat about this next week.